Source/WebCore/css/html.css

```
@@ -853,6 +853,10 @@ input:-webkit-autofill, input:-webkit-autofill-strong-password, input:-webkit-au
     color: #000000 !important;
 }
 
+:-webkit-autofill {
```

```suggestion
:autofill {
```

WebKit supports the unprefixed version fwiw: https://searchfox.org/wubkat/rev/cff04f81eb10bd7f7b8fad6204252122612c94e6/Source/WebCore/css/CSSPseudoSelectors.json#87

---

Ah I didn't realise that, just saw the prefix in the stylesheet and assumed it didn't. I'll swap it over.